src/components/arrival_time/index.js

```diff
@@ -6,6 +6,14 @@ var ArrivalTime = BaseIcon.extend({
 
   defaultUnit: 'Uhr',
 
+    constructor: function(options) {
+
+      if (typeof options.defaultUnit !== 'undefined') this.defaultUnit = options.defaultUnit;
+
+        this.base(options);
+
+    },
+
```

Comment:
Good find! 👍

But you should check if `options` are actually passed to the constructor, to prevent errors. Something like `options || options = {}` before working with them.

Reply:
Done